jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnectionFactory.java

```diff
@@ -64,12 +64,25 @@ public List<String> getProtocols()
     @ManagedAttribute("The buffer size used to read from the network")
     public int getInputBufferSize()
     {
-        return _inputbufferSize;
+        return _inputBufferSize;
     }
 
     public void setInputBufferSize(int size)
     {
-        _inputbufferSize = size;
+        _inputBufferSize = size;
+    }
+
+    public boolean isUseVirtualThreadToInvokeRootHandler()
+    {
+        return _useVirtualThreadToInvokeRootHandler;
+    }
+
+    public void setUseVirtualThreadToInvokeRootHandler(boolean useVirtualThreadToInvokeRootHandler)
```

Comment:
Generally speaking, I'm not liking all these extra setters on Connectors.
I think (specially for Jetty-12), we need to step back and reconsider how we configure these kinds of things.